<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWebView" name="cam">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonHaut">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonBas">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>520</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonGauche">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonDroite">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camversleHaut">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camversleBas">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>520</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camverslaDroite">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camverslaGauche">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disconnexion">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AdresseIp">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.106</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Caméra</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>430</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>20</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>118</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Caméra</string>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>118</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>573</x>
      <y>430</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vitesse">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>240</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="speedL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="speedR">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cpt1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cpt2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
